testing/pom.xml

```diff
@@ -359,5 +353,19 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>dse</id>
```

Comment:
Could it break the default "it" profile selection? AFAIK, if any profile is specified _explicitly_ all the "default" profiles will not be active anymore :thinking:

Reply:
Might be worth refactoring those `it-*` profiles into plain `cassandra-3.11`, `cassandra-4.0` and `dse-6.8`... and reuse them both for compilation and testing. WDYT?

As for the `-6.8` suffix I think it matches the persistence modules name well (it also has the version suffix).

Reply:
> Could it break the default "it" profile selection? AFAIK, if any profile is specified explicitly all the "default" profiles will not be active anymore

From what I've seen so far, no.

> Might be worth refactoring those it-* profiles into plain cassandra-3.11, cassandra-4.0 and dse-6.8... and reuse them both for compilation and testing. WDYT?

I think that might be a bit overkill. I'd prefer to be able to build as much as possible without profiles.